src/components/Metrics.js

```diff
@@ -15,7 +22,25 @@ const Metrics = React.memo(function Metrics({
 }) {
   const { layoutName } = useLayout()
   const compactMode = layoutName === 'small'
-  const { requestToken, stakeToken } = useAppState()
+  const { requestToken, stakeToken, currencies } = useAppState()
+  const [currencyIndex, setCurrencyIndex] = useState(0)
+  const currency = useMemo(() => {
+    if (!currencies.length) {
```

Comment:
maybe we can always have the usd currency right? i mean return always with the USD as default